src/site/content/en/blog/cls-web-tooling/index.md

```diff
@@ -0,0 +1,278 @@
+        <td>1 June 2021</td>
+        <td>After 1 June 2021, available as
+        <code>experimental_uncapped_cumulative_layout_shift</code>
+        December 14th, 2021</td>
+      </tr>
+      <tr>
+    </tbody>
+  </table>
+</div>
+
+For most developers,
+this change is expected to happen seamlessly with no action needed to take advantage of data from the fix.
+
+The update to CLS has also been made to [Search Console](https://search.google.com/search-console/about) and will be reflective from 1 June, 2021.
+
+
+## "Old" CLS
+
+As some developers may wish to monitor the older definition of CLS alongside the windowing-adjustment,
+we have good news to share: we are exposing "old CLS" in Lighthouse and CrUX.
+
+In Lighthouse v8,
+it's available in the JSON as
+`audits['cumulative-layout-shift'].details.items[0].totalCumulativeLayoutShift`.
+
+You'll find it as
+`experimental_uncapped_cumulative_layout_shift`
+in the CrUX API and as
+`experimental.uncapped_cumulative_layout_shift` in CrUX BigQuery.
+
+After June 1st,
+[the following CrUX API request](https://developers.google.com/web/tools/chrome-user-experience-report/api/reference/rest/v1/records/queryRecord?apix_params=%7B%22resource%22%3A%7B%22origin%22%3A%22https%3A%2F%2Fweb.dev%22%2C%22metrics%22%3A%5B%22experimental_uncapped_cumulative_layout_shift%22%5D%7D%7D)
+will return the "old CLS" metric:
+
+```js
+{
+  "origin": "https://web.dev",
+  "metrics": [
+    "experimental_uncapped_cumulative_layout_shift"
+  ]
+}
+```
+
+After June 8, the following
+[CrUX BigQuery](https://developers.google.com/web/tools/chrome-user-experience-report/bigquery/getting-started)
+will compare old and new CLS:
+
+```sql
+WITH
+  new_data AS (
+  SELECT
+    cls
+  FROM
+    `chrome-ux-report.all.202105`,
+    UNNEST(layout_instability.cumulative_layout_shift.histogram.bin) AS cls
+  WHERE
+    origin = 'https://web.dev'
+    AND effective_connection_type.name = '4G'
+    AND form_factor.name = 'phone'),
+  old_data AS (
+  SELECT
+    uncapped_cls
+  FROM
+    `chrome-ux-report.all.202105`,
+    UNNEST(experimental.uncapped_cumulative_layout_shift.histogram.bin) AS uncapped_cls
+  WHERE
+    origin = 'https://web.dev'
+    AND effective_connection_type.name = '4G'
+    AND form_factor.name = 'phone')
+SELECT
+  cls.start AS start,
+  cls.`END` AS `end`,
+  cls.density AS cls_density,
+  uncapped_cls.density AS uncapped_cls_density
+FROM
+  new_data
+INNER JOIN
+  old_data
+ON
+  new_data.cls.start = old_data.uncapped_cls.start
+```
+
+You will be able to continue relying on this data to be present for a period of up to 6 months,
+with uncapped CLS being retired on December 14th, 2021.
+
+## Updating the CLS weighting in Lighthouse
+
+When CLS was first introduced in Lighthouse,
+it was a brand new sparkly metric.
+As such, in order to make sure developers had time to test, benchmark,
+and optimize against it,in the performance score CLS was weighed less heavily.
```

Comment:
This line is awkward (also there's a missing space after the comma). Can we reword to this?

```suggestion
and optimize against it, CLS was weighed less heavily in the performance score.
```